adapters/http4s/src/main/scala/caliban/Http4sAdapter.scala

```diff
@@ -90,22 +102,29 @@ object Http4sAdapter {
   def executeRequest[R0, R, E](
     interpreter: GraphQLInterpreter[R, E],
     provideEnv: R0 => R,
-    skipValidation: Boolean = false
+    skipValidation: Boolean = false,
+    enableIntrospection: Boolean = true
   ): HttpApp[RIO[R0, *]] =
     Kleisli { req =>
       object dsl extends Http4sDsl[RIO[R0, *]]
       import dsl._
       for {
-        query    <- req.attemptAs[GraphQLRequest].value.absolve
-        result   <- executeToJson(interpreter, query, skipValidation).provideSome[R0](provideEnv)
+        query <- req.attemptAs[GraphQLRequest].value.absolve
+        result <- executeToJson(
+                   interpreter,
+                   query,
+                   skipValidation = skipValidation,
+                   enableIntrospection = enableIntrospection
+                 ).provideSome[R0](provideEnv)
         response <- Ok(result)
       } yield response
     }
 
   def makeWebSocketService[R, E](
     interpreter: GraphQLInterpreter[R, E],
     skipValidation: Boolean = false,
-    keepAliveTime: Option[Duration] = None
+    keepAliveTime: Option[Duration] = None,
+    enableIntrospection: Boolean = true
```

Comment:
This is super-nitpicking but can you put it right after `skipValidation`, before `keepAliveTime`. It feels better to have them next to each other 😄 Same for the uzhttp one.

Reply:
I actually did this intentionally in case someone had used positional arguments. Happy to move it.

Reply:
`keepAliveTime` was just added a few days ago so I think it's fine (and the types don't match so no risk).